metrics_utils/src/lib.rs

```diff
@@ -1267,3 +1270,98 @@ impl BatchMintPersisterMetricsConfig {
         );
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct RedisReceiverMetricsConfig {
+    start_time: Gauge,
+    transactions_received: Counter,
+    accounts_received: Counter,
+    transactions_parsed: Counter,
+    accounts_parsed: Counter,
+    transaction_parse_errors: Counter,
+    account_parse_errors: Counter,
+}
+
+impl Default for RedisReceiverMetricsConfig {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl RedisReceiverMetricsConfig {
+    pub fn new() -> Self {
```

Comment:
Why do we need `new()` that accepts zero parameters and is a full copy of `Default`? Couldn't we use `RedisReceiverMetricsConfig::default()` where `RedisReceiverMetricsConfig::new()` is requested and that's it?

Reply:
tbh because this seems to be the code standard in the repo

Reply:
Then it's an odd standard we should get rid of, imo. This probably deserves a dedicated ticket.